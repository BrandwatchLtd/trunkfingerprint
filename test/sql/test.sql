create schema foo;
create extension trunkfingerprint schema foo;

create table results(run int, fingerprint text);

insert into results values (
       1,
       foo.get_db_fingerprint(
              0,
              null,
              null,
              '{}',
              '{"(public,results)"}',
              '{}'
       )
);

create table bar(baz serial);

insert into results values (
       2,
       foo.get_db_fingerprint(
              0,
              null,
              null,
              '{}',
              '{"(public,results)"}',
              '{}'
       )
);

drop table bar;

insert into results values (
       3,
       foo.get_db_fingerprint(
              0,
              null,
              null,
              '{}',
              '{"(public,results)"}',
              '{}'
       )
);

select array_agg(run)
from results
group by fingerprint
order by 1;
