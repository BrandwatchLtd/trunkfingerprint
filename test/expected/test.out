create extension trunkfingerprint;
create table results(run int, fingerprint text);
insert into results values (
       1,
       trunkfingerprint.get_db_fingerprint(
              0,
              null,
              null,
              '{}',
              '{"(public,results)"}',
              '{}'
       )
);
create table bar(baz serial);
insert into results values (
       2,
       trunkfingerprint.get_db_fingerprint(
              0,
              null,
              null,
              '{}',
              '{"(public,results)"}',
              '{}'
       )
);
drop table bar;
insert into results values (
       3,
       trunkfingerprint.get_db_fingerprint(
              0,
              null,
              null,
              '{}',
              '{"(public,results)"}',
              '{}'
       )
);
select array_agg(run)
from results
group by fingerprint
order by 1;
 array_agg 
-----------
 {1,3}
 {2}
(2 rows)

